src/base/static/components/organisms/place-list.js

```diff
@@ -254,7 +254,7 @@ PlaceList.propTypes = {
 };
 
 const mapStateToProps = state => ({
-  places: filteredPlacesSelector(state),
+  places: filteredPlacesSelector(state).filter(place => !place.private),
```

Comment:
This filter is only to prevent newly-submitted private places from rendering. We should be able to remove this when we refactor the models into our ducks and no longer have to rely on Backbone for network requests.